backend/src/models/userModel.ts

```
@@ -47,7 +47,7 @@ export const createUser = async ({
     });
 
     return newUser;
-  } catch (error) {
+  } catch {
     throw new Error();
   }
```

This is what we call a "useless catch." To catch and rethrow without doing anything means that you should not have this try/catch at all. In general, only catch when you can recover from the error gracefully with some kind of fallback, otherwise, just let it crash. But a catch and rethrow is like doing nothing at all.